pkg/client/client.go

```diff
@@ -1127,7 +1146,7 @@ func (c *Client) WaitForDeploymentRollout(ctx context.Context, dep *appsv1.Deplo
 			return false, nil
 		}
 		if d.Status.UpdatedReplicas != d.Status.Replicas {
-			lastErr = errors.Errorf("the number of pods targeted by the deployment (%d pods) is different "+
+			lastErr = errors.Errorf("the number of pods targeted by the deployment (%d pods) is diff`<erent "+
```

Comment:
Looks like a typo here
```suggestion
			lastErr = errors.Errorf("the number of pods targeted by the deployment (%d pods) is different "+
```

Reply:
However, I do wonder if something like the `generation`-al changes will end up passing through this, still triggering the update.

Reply:
sorry, must have fat-fingered those in while scrolling in my editor.  Change dropped with a80e176 -> fa747c9.